### DISCUSSION TITLE
Mapping of Expressions

### DISCUSSION THREAD

**Discussion by Aayu261** [soot-oss/SootUp#1133] [Q&A]
Is there any API to get mapping of jimple expressions to the actual expressions on which the analysis is done?
Attached below is jimple representation of expressions of how actually it looks like :
[image](https://github.com/user-attachments/assets/c36f1c81-3c64-4035-a861-8c88a8bca837)

**Comment by swissiety** [comment]
to get an impression you can look at the tests in MiniTestSuite
